pallets/dkg-proposal-handler/src/lib.rs

```diff
@@ -117,15 +177,52 @@ impl<T: Config> Pallet<T> {
 		};
 
 		let mut sig = [0u8; 65];
-		let mut msg = [0u8; 32];
 		sig[0..32].copy_from_slice(&sig_r[..]);
 		sig[32..64].copy_from_slice(&sig_s[..]);
 		sig[64] = sig_v;
+		let mut msg = [0u8; 32];
 		msg.copy_from_slice(&msg_hash[..]);
 
-		return match sp_io::crypto::secp256k1_ecdsa_recover(&sig, &msg) {
-			Ok(_) => Ok(()),
-			Err(_) => Err(DispatchError::BadOrigin),
-		}
+		return sp_io::crypto::secp256k1_ecdsa_recover(&sig, &msg).is_ok()
+	}
+
+	fn validate_proposal_signature(data: &Vec<u8>, signature: &Vec<u8>) -> bool {
+		let mut sig = [0u8; 65];
```

Comment:
Can we use a constant like `PROPOSAL_SIGNATURE_LENGTH` here instead of using just `65`